src/gdash/scripts/_contains/_contains.gml

```diff
@@ -15,9 +15,10 @@ _contains("hello", "ello");
 // => true
 */
 
-var collection = argument[0];
-var target = argument[1]
-var fromIndex = 0;
+var 

```

Comment:
Alright, yeah. I would like to keep the one-var-per-line style. As for time, I think waiting a little bit would be better than merging immediately, since this is pretty sweeping. I'll be writing more tests to ensure there's good coverage of expected behavior so I can test this branch against it. 100% test coverage is a goal for this repo anyway.